expected/16.0/extra/bool.out

```diff
@@ -4,7 +4,7 @@
 CREATE EXTENSION sqlite_fdw;
 --Testcase 001:
 CREATE SERVER sqlite_svr FOREIGN DATA WRAPPER sqlite_fdw
-OPTIONS (database '/tmp/sqlitefdw_test.db');
+OPTIONS (database '/tmp/sqlite_fdw_test/common.db');
```

Comment:
Could you explain about this change?

Reply:
It's for 
- simpler test script, just `testdir='/tmp/sqlite_fdw_test'; rm -rf "$testdir";`
- less chaos in `/tmp`, because I have many programs with files in `/tmp`
- up to 1.5% faster tests, maybe because `/tmp/sqlite_fdw_test` inodes can be better cached than `/tmp` with many different files from different processes. Also `/tmp/sqlite_fdw_test` content have no changes not from `sqlite_fdw` during testing.

Reply:
Thank you. Please help me resolve this thread.